oomcli/cmd/get_meta_entity.go

```diff
@@ -14,28 +14,33 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var listEntityCmd = &cobra.Command{
+var getMetaEntityCmd = &cobra.Command{
 	Use:   "entity",
-	Short: "list all existing entities",
+	Short: "get existing entity given specific confitions",
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
 		oomStore := mustOpenOomStore(ctx, oomStoreCfg)
 		defer oomStore.Close()
 
 		entities, err := oomStore.ListEntity(ctx)
 		if err != nil {
-			log.Fatalf("failed listing entities, error %v\n", err)
+			log.Fatalf("failed getting entities, error %v\n", err)
 		}
 
+		if len(args) > 0 {
+			entities = entities.Filter(func(e *types.Entity) bool {
+				return e.Name == args[0]
+			})
+		}
 		// print entities to stdout
-		if err := printEntities(entities, *listOutput); err != nil {
+		if err := printEntities(entities, *getOutput); err != nil {
```

Comment:
consider using `getMetaOutput`?

Reply:
Is the output optional the same for` get online` and `get meta`? If they are the same, `getOutput` should be used, otherwise, `getMetaOutput` should be used.

Reply:
hmm not sure if we want `Column` for `get online`

Reply:
The `getMetaOutout` is used